src/js/media-time-range.js

```diff
@@ -408,37 +415,35 @@ class MediaTimeRange extends MediaChromeRange {
   }
 }
 
-function getBoxPosition(el, box, percent) {
-  const rect = el.getBoundingClientRect();
-  const rangeRect = el.range.getBoundingClientRect();
+function getBoxPosition(el, box, ratio) {
+  let position = `${ratio * 100}%`;
+
+  // Get the element that enforces the bounds for the time range boxes.
+  const bounds =
```

Comment:
it's probably worth moving this back down, if we return early on line 436 `if !boxWidth`, we won't need to do the work for these bounds.

Reply:
👍 will do

Reply:
@gkatsev which example? 
on my end it seems to work.

Reply:
@luwes with https://media-chrome-git-fork-luwes-fix-range-bugs-mux.vercel.app/examples/advanced.html, when hovering over the progress bar I get a bunch of these errors in the console:
```
Uncaught DOMException: CSSStyleSheet.cssRules getter: Not allowed to access cross-origin stylesheet
```
from `getOrInsertCSSRule` in `element-utils.js`
However, it works fine on https://media-chrome-mux.vercel.app/examples/advanced.html

Reply:
what's your Firefox version? gonna try to repro. strange because I'm not doing much different than before.
maybe 1 thing is that 1 of the style tags is now in the container element instead of the shadowRoot

Reply:
Seeing the same behavior in my Firefox for Windows, but a quick test in Browserstack with Firefox 105 and 106 (Beta) shows it working as expected 🤔

Reply:
ah, I do have Dark Reader enabled, let me check. Though, it's weird that it's working in main but not with this fork

Reply:
true yes weird, let me try install that too

Reply:
@luwes that was it. I guess I had dark reader disabled on main, but not on your fork preview.

Reply:
So, yeah, not a blocker for this PR. Probably worth investigating but as a low priority thing.

Reply:
will add in a try / catch to workaround this issue